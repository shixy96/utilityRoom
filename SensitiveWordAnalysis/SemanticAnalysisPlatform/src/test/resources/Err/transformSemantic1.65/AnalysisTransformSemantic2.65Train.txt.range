x
-1 1
1 0 12
2 0 14.75
3 0 24.75
4 0 17.199999999999999
5 0 18.5625
6 0 15
7 0 24
8 0 27.449999999999999
9 0 24
10 0 25.5
11 0 21.75
12 0 25.5
13 0 16.439999999999998
14 0 21.399999999999999
15 0 17.450000000000003
16 0 19.5
17 0 18.149999999999999
18 0 13.5
19 0 18.149999999999999
20 0 3
21 0 3
