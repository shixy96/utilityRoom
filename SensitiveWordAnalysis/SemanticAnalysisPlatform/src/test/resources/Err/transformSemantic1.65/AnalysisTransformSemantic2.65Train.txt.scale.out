log2c=5 log2g=-7 rate=81.7056
log2c=-1 log2g=-7 rate=73.6072
log2c=5 log2g=-1 rate=85.4093
log2c=-1 log2g=-1 rate=82.658
log2c=11 log2g=-7 rate=81.8114
log2c=11 log2g=-1 rate=87.009
log2c=5 log2g=-13 rate=73.5636
log2c=-1 log2g=-13 rate=74.5969
log2c=11 log2g=-13 rate=81.7118
log2c=-3 log2g=-7 rate=74.5969
log2c=-3 log2g=-1 rate=82.0915
log2c=-3 log2g=-13 rate=74.5969
log2c=5 log2g=1 rate=87.4448
log2c=-1 log2g=1 rate=86.1625
log2c=11 log2g=1 rate=87.2456
log2c=-3 log2g=1 rate=84.7308
log2c=9 log2g=-7 rate=81.7491
log2c=9 log2g=-1 rate=87.0215
log2c=9 log2g=-13 rate=81.662
log2c=9 log2g=1 rate=87.5755
log2c=5 log2g=-11 rate=80.5353
log2c=-1 log2g=-11 rate=74.5969
log2c=11 log2g=-11 rate=81.7305
log2c=-3 log2g=-11 rate=74.5969
log2c=9 log2g=-11 rate=81.7056
log2c=3 log2g=-7 rate=81.6931
log2c=3 log2g=-1 rate=84.8988
log2c=3 log2g=-13 rate=74.5969
log2c=3 log2g=1 rate=87.6439
log2c=3 log2g=-11 rate=73.5636
log2c=5 log2g=-5 rate=81.7678
log2c=-1 log2g=-5 rate=81.0644
log2c=11 log2g=-5 rate=83.3427
log2c=-3 log2g=-5 rate=73.6632
log2c=9 log2g=-5 rate=82.8634
log2c=3 log2g=-5 rate=81.7367
log2c=15 log2g=-7 rate=82.4463
log2c=15 log2g=-1 rate=86.9094
log2c=15 log2g=-13 rate=81.3383
log2c=15 log2g=1 rate=86.7476
log2c=15 log2g=-11 rate=81.7927
log2c=15 log2g=-5 rate=84.8491
log2c=5 log2g=-15 rate=74.5969
log2c=-1 log2g=-15 rate=74.5969
log2c=11 log2g=-15 rate=81.6682
log2c=-3 log2g=-15 rate=74.5969
log2c=9 log2g=-15 rate=80.554
log2c=3 log2g=-15 rate=74.5969
log2c=15 log2g=-15 rate=81.3383
log2c=-5 log2g=-7 rate=74.5969
log2c=-5 log2g=-1 rate=82.1786
log2c=-5 log2g=-13 rate=74.5969
log2c=-5 log2g=1 rate=82.6766
log2c=-5 log2g=-11 rate=74.5969
log2c=-5 log2g=-5 rate=74.5969
log2c=-5 log2g=-15 rate=74.5969
log2c=5 log2g=3 rate=87.8245
log2c=-1 log2g=3 rate=87.4012
log2c=11 log2g=3 rate=87.4261
log2c=-3 log2g=3 rate=86.038
log2c=9 log2g=3 rate=87.6066
log2c=3 log2g=3 rate=88.1108
log2c=15 log2g=3 rate=86.9592
log2c=-5 log2g=3 rate=84.5689
log2c=7 log2g=-7 rate=81.7554
log2c=7 log2g=-1 rate=86.6231
log2c=7 log2g=-13 rate=80.5415
log2c=7 log2g=1 rate=87.5257
log2c=7 log2g=-11 rate=81.6122
log2c=7 log2g=-5 rate=81.9172
log2c=7 log2g=-15 rate=73.5636
log2c=7 log2g=3 rate=87.756
log2c=5 log2g=-9 rate=81.6682
log2c=-1 log2g=-9 rate=74.5969
log2c=11 log2g=-9 rate=81.6558
log2c=-3 log2g=-9 rate=74.5969
log2c=9 log2g=-9 rate=81.6807
log2c=3 log2g=-9 rate=80.5353
log2c=15 log2g=-9 rate=81.6433
log2c=-5 log2g=-9 rate=74.5969
log2c=7 log2g=-9 rate=81.7242
log2c=1 log2g=-7 rate=80.6287
log2c=1 log2g=-1 rate=83.6975
log2c=1 log2g=-13 rate=74.5969
log2c=1 log2g=1 rate=87.1646
log2c=1 log2g=-11 rate=74.5969
log2c=1 log2g=-5 rate=81.7989
log2c=1 log2g=-15 rate=74.5969
log2c=1 log2g=3 rate=88.0361
log2c=1 log2g=-9 rate=73.5636
log2c=5 log2g=-3 rate=83.3302
log2c=-1 log2g=-3 rate=81.8238
log2c=11 log2g=-3 rate=84.6872
log2c=-3 log2g=-3 rate=81.6184
log2c=9 log2g=-3 rate=84.4507
log2c=3 log2g=-3 rate=82.1289
log2c=15 log2g=-3 rate=86.2247
log2c=-5 log2g=-3 rate=75.4809
log2c=7 log2g=-3 rate=83.9216
log2c=1 log2g=-3 rate=81.8176
log2c=13 log2g=-7 rate=82.1849
log2c=13 log2g=-1 rate=87.0588
log2c=13 log2g=-13 rate=81.6495
log2c=13 log2g=1 rate=86.8347
log2c=13 log2g=-11 rate=81.6931
log2c=13 log2g=-5 rate=83.8967
log2c=13 log2g=-15 rate=81.5313
log2c=13 log2g=3 rate=87.2518
log2c=13 log2g=-9 rate=81.7367
log2c=13 log2g=-3 rate=85.5462
